gateway/webhook/webhook_test.go

```diff
@@ -433,3 +433,51 @@ func TestRecordWebhookErrors(t *testing.T) {
 	})
 	require.EqualValues(t, m.LastValue(), 1)
 }
+
+func TestPrepareRequestBody(t *testing.T) {
```

Comment:
Do we need the other scenarios as well: https://github.com/rudderlabs/rudder-server/pull/4677#discussion_r1601463887

Reply:
we had these 4 only

Reply:
In the table-driven test, we wrote, I see only 3 sub-tests that's why I asked.